SimpleJavaProject/src/main/java/dmitrii_tofan/_HW_3/Break_Continue_Main.java

```diff
@@ -0,0 +1,33 @@
+package dmitrii_tofan._HW_3;
+
+import java.util.Enumeration;
+import java.util.Scanner;
+
+public class Break_Continue_Main {
+    public static void main(String[] args) {
+
+// 1.Create a program, which will ask user to enter a number, if number will be negative exit the loop
+
+        Scanner sc = new Scanner(System.in);
+        int sum = 0;
+        for (int i = 0; i < 10; i++) {
```

Comment:
Here should be a while loop, because in for you will ask max 10 times.
But you have to ask as many times as users enter a positive number.